public/kubernetes-guides/advanced-guides/hpa.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Horizontal Pod Autoscaling"
+---
+
+import { version } from '/snippets/custom-variables.mdx';
+
+The Horizontal Pod Autoscaler (HPA) automatically scales Kubernetes workloads based on resource utilization or other defined metrics.
+
+In Talos, HPA relies on the [Metrics Server ](../monitoring-and-observability/deploy-metrics-server)to collect data from the kubelets in your cluster. It then uses these metrics to decide when and how to scale your workloads.
+
+This guide walks you through how to use HPA to scale workloads on a Talos cluster.
+
+## Before you begin
+
+You’ll need the following to use HPA in your Talos cluster:
+
+* **A running Talos cluster**: If you don’t have one yet, see the <a href={`../../talos/${version}/getting-started/getting-started`}>Getting Started</a> or <a href={`../../talos/${version}/getting-started/prodnotes`}>Production Cluster</a> guides to create a cluster.
+
+* **kubectl configured for your cluster**: Verify your setup by running:
+	```bash
+	kubectl get nodes
+	```
+
+* **Metrics Server deployed**: Make sure the Metrics Server is running in your cluster. See the [Deploy Metrics Server](../monitoring-and-observability/deploy-metrics-server) guide for instructions.
```

Comment:
I _think_ the metrics server may also need the a certificate signing request approver because the nodes will get new certificates and if they're not approved the metric endpoints wont be available. https://github.com/postfinance/kubelet-csr-approver Maybe this is something we should document somewhere else. I only found reference to it in the troubleshooting https://docs.siderolabs.com/talos/v1.11/troubleshooting/troubleshooting#talos-complains-about-node-not-found